.vib/consul/goss/goss.yaml

```diff
@@ -33,11 +23,3 @@ command:
       - "consul-1"
       - "consul-2"
     stderr: []    
```

Comment:
I think you can remove this as it does not add any value. This will succeed if either there is text in `stderr` or not.

Reply:
I didn't know that, I thought it could be used as debug. Removed, thanks!